### PR TITLE
Making it work

### PR DESCRIPTION
**Pull request by sassanh**

### CODE REVIEW

File: src/tests/integrity.test.ts [hunk 1]
```diff
@@ -41,6 +36,27 @@ describe('Normal flow', (): void => {
             .to.be.instanceOf(SequelizeUser)
             .and.to.have.property('id')
             .that.is.a('number')
-            .that.is.equal(secondUser.id);
+            .and.is.equal(secondUser.id);
+    });
+
+    it('should be able to create a user, issue a token for it and' +
+        ' authenticate the token', async (): Promise<void> => {
```
Comment (hesamd) [inline src/tests/integrity.test.ts:52]: looks like extra space in the beginning of the string

File: src/tests/integrity.test.ts [hunk 2]
```diff
@@ -41,6 +36,27 @@ describe('Normal flow', (): void => {
...
+        const {SequelizeAccessToken, SequelizeUser} = models;
+        const user = new SequelizeUser({username: 'a', isActive: true});
```
Comment (hesamd) [inline src/tests/integrity.test.ts]: In tests, my preferece is using real world example assigned to a const variable e.g. USER = 'User1!3m' or even using unicode characters if you support them. `a` and `b` can miss some of the potential bugs, etc.

**Review by hesamd — CHANGES_REQUESTED** [review]

File: src/tests/integrity.test.ts (hunk 1 above)
Comment (sassanh) [reply src/tests/integrity.test.ts:52]: @hesamd 
It's appending to the string in previous line, so we need it.

File: src/tests/integrity.test.ts (hunk 2 above)
Comment (sassanh) [reply src/tests/integrity.test.ts]: @hesamd done, in the new commit I used faker to generate test values.

**Review by hesamd — APPROVED** [review]

**Pull request merged**